api/v1beta1/octaviaapi_types.go

```diff
@@ -141,6 +141,10 @@ type OctaviaAPISpecCore struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// TLS - Parameters related to the TLS
 	TLS OctaviaApiTLS `json:"tls,omitempty"`
+
+	// +kubebuilder:validation:Optional
```

Comment:
1. here it's defined as Optional

Reply:
yeah, to keep option to be passed by spec otherwise we trust on the octavia_api one and it will be passed by the controller